indico/modules/rb_new/client/js/common/bookings/BookingObjectLink.jsx

```diff
@@ -80,7 +80,7 @@ export default class BookingObjectLink extends React.PureComponent {
             contribution: Translate.string('This booking is linked to a contribution:'),
             sessionBlock: Translate.string('This booking is linked to a session block:'),
         };
-        return canAccess && (
+        return canManage && (
```

Comment:
for this one access is fine (that's when viewing a booking)

Reply:
Not always, it could be right before linking (book a room). I guess I need to check `pending` also.

Reply:
I think that's an edge case which is OK-ish since you won't get there except by messing with the link. The whole point of these access checks is to:

- not allow someone to link an event they don't manage
- not disclose information about events they cannot see otherwise

being able to see information they could access anyway isn't a problem

Reply:
Fair enough, updated.